docs/sphinx/manuals/roc_recv.rst

```diff
@@ -65,6 +66,13 @@ Driver
 
 If the driver is omitted, some default driver is selected. If the user did specify the output and it is a file with a known extension, the appropriate file driver is selected. Otherwise, the first device driver available on the system is selected.
 
+Multicast
+---------
+
+*IFACE_IP* should be an IP address of the network interface on which to join to the multicast group. It may be "0.0.0.0" or "[::]" to join the multicast group on all available interfaces.
+
+If *IFACE_IP* is not specified and multicast address is used, user should manually join to the multicast group.
```

Comment:
`to join TO the multicast group` -> `to join the multicast group`

I'd also clarify:

`"0.0.0.0" or "[::]"` -> `"0.0.0.0" (for IPv4) or "[::]" (for IPv6)`

`user should manually join TO the multicast group` -> `THE user is responsible for joining the multicast group manually`

Reply:
Done